README.md

```diff
@@ -1,12 +1,455 @@
 # Hydra.NET
 
-Hydra.NET is a simple library that provides building blocks for creating hypermedia-driven web APIs with the [Hydra specification](https://www.hydra-cg.com/spec/latest/core/).
+Hydra.NET is a simple library for ASP.NET that provides the building blocks necessary for creating hypermedia-driven Web APIs with the [Hydra specification](https://www.hydra-cg.com/spec/latest/core/).
 
-## Usage
+## Quickstart
```

Comment:
Great work! Question: what do you think about creating another repo for the ASP.NET quickstart that includes code? We could link to it at the beginning of this README and it would provide a pattern for creating additional quickstarts that use different frameworks.

Reply:
I like that idea. I can start putting the new repo together or I can also add to something you create.

Reply:
Feel free to put it together!